### DISCUSSION TITLE
How Much Longer?

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#607] [Q&A]
I know this is the umpteenth time I've made a post asking when update 2.0 will be coming, and I'm probably gonna get more comments telling me to be patient, and no one who has the beta test version can say what the new update has in it, but how much longer will we have to wait for version 2.0? I've been waiting so long for the update, and I don't know how much longer I can wait for the update. A ballpark estimate of when it should be here would be good enough for the time being.

**Comment by ArtsyStar1** [comment]
Don’t get me wrong, I do believe it’s weird how long it’s taking for the 2.0 update but keep in mind that it’s pretty much just one person working on toonsquid so I can see why it’s taking a while to be fully finished.

**Comment by budaloops** [comment]
It takes time

**Comment by andydjames** [comment]
Big shout out to Keiwan for all his hard work and vision for ToonSquid. 
Ten bug fix releases to make sure the app's as solid as it can be before rushing out version 2.0. 
Respect, man.

**Comment by joelmayer87** [comment]
"I don't know how much longer i can wait"

Like... are you catching a train or something? :D 

It's one guy literally doing this in his spare time, and when 2.0 is out you can't wait for 3.0 etc. etc. As somebody who started animation on paper and a Super 8 camera and later Flash 4 and a Wacom Graphire Small, don't wait for any perfection in technology, there isn't and there never will be. Make due with what tools you have TODAY.

Because there's great animators using very simple tools and mediocre animators using very advanced tools. Sure, updates are nice, i had my own set of feature requests but mainly i'm just animating and use what i have. I GUARANTEE you, Toonsquid 2.0 won't make your animation any better, it might bring some small quality of life stuff and make it even more enjoyable to use but again, that we can animate on a tablet in this capacity at all right now is great already.

Toonsquid is really potent as it is and i'm sure 2.0 will come eventually and will be great as well but that shouldn't hold anyone back from just MAKING things right now.

**Reply by DWrightAnimation** [reply]
I'm gonna let you in on something, I've been doing an animation summer camp thing on my YouTube channel, and I wanted to teach rigging in one of the lessons. Unfortunately, with 2.0 coming out any day now, I haven't been able to make the videos for the rigging lesson out of fear that the 2.0 update will change things radically, making the videos obsolete. I know I can create anything with what I currently have, I just need the update to come soon so my rigging videos will be secure for a long time. In the meantime, I've taught the 12 principles of animation, and a little lesson on story structure, with other lessons coming in the coming weeks.

**Reply by joelmayer87** [reply]
Thanks for telling me your secrets. I made a two hour Moho tutorial and a few months later they released a new version. It still applies. Adam Philips wrote a huge book about Toon Boom Harmony, few months later, they renamed the whole line-up and released a new version and yet his book still applies. My argument still stands, don't wait for any software to catch up to what you want to do.

**Reply by ll-ghin** [reply]
Just make your videos, and Keiwando can work at his own pace.  You shouldn't wait on someone to make something nor should your desires affect how hard or how much he works.

it is weird to constantly make threads about this.

**Comment by CowlBlue** [comment]
He is likely working a full time job and doing toonsquid dev in his spare time.  If you understand what that's like than you would know that your tone comes across as entitled and maybe discouraging.  I think there's a better way to ask for written updates/sneak peeks etc.
